examples/run-all.sh

```diff
@@ -1,33 +1,61 @@
 #!/bin/bash
 set -e -u
 
-# Print the problmatic case and increase error count
-detect_error(){
-    echo "Error while running case ${array[1]}"
+if ! command -v precice-aste-run &> /dev/null || ! command -v precice-aste-evaluate &> /dev/null; then
+    echo "Error: It seems that ASTE is not installed or not discoverable in your PATH."
```

Comment:
You can always refer to the ASTE installation instructions as a hint. You don't need to be too specific in the error message.

Reply:
Alright, thanks :+1: